src/components/FirebaseAuth.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+
+import firebase from 'firebase';
+import StyleFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+
+firebase.initializeApp({
+  apiKey: 'AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA',
+  authDomain: 'dreamerscholars.firebaseapp.com',
+});
+
+class FirebaseAuth extends Component {
```

Comment:
It's not clear from the name what this is supposed to be (`Auth` doesn't mean much visually), maybe `FirebaseLoginScreen`?

Reply:
ok changed